docs/vendor-docs/stripe.md

```
@@ -0,0 +1,126 @@
+
+---
+
+## API Version History
+
+| Date | API Version | Impact | Notes |
+|------|-------------|--------|-------|
+| 2026-01-28 | `2026-01-28.clover` | Current | Our pinned version |
+| 2025-03-31 | `2025-03-31.basil` | **BREAKING** | `current_period_end` moved to items |
+
+**How to check current version:**
+```bash
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)